src/stories/Heading.stories.js

```diff
@@ -38,7 +38,7 @@ const Headings = {
 
 const Template = (args) => (
 	<Heading {...args} variant={args.variant}>
-		I'm a h{args.variant}
+		I'm a h{args.variant}. Structural Engineering 
```

Comment:
Is the Structural Engineering part necessary to keep?

Reply:
It is not. I removed it